src/js/RateCurve.js

```diff
@@ -89,7 +89,7 @@ RateCurve.prototype.rcCommandRawToDegreesPerSecond = function (rcData, rate, rcR
 
         var inputValue = this.rcCommand(rcData, rcRate, deadband);
         var maxRc = 500 * rcRate;
-        
+        var absRc = Math.abs(inputValue) / maxRc;
```

Comment:
In the firmware, this variable is called `rcCommandfAbs`. Can you please rename it to be the same? In this way is easier to compare codes if some change is needed in a future.

Reply:
@McGiverGim Sure i can do that. Maybe we should rename ``inputValue`` to ``rcCommandf`` also?
I also think ``inputValue`` should be scaled down when it is first assigned a value instead of doing it with ``var rcInput = inputValue / maxRc;``. Then we could get rid of ``rcInput`` and just use ``inputValue`` like in the betaflight code.

Reply:
I don't know what others will think, to me, if you can "refactor" the code to be near the firmware one, will be perfect. You only need to have caution with code that executes for older versions.

Reply:
@McGiverGim Just pushed a new commit with the changes i described. The functionality and overall structure is the same as before.